src/include/execution/sql/storage_interface.h

```diff
@@ -95,6 +95,14 @@ class EXPORT StorageInterface {
    */
   bool IndexInsertUnique();
 
+  /**
+   * Insert into the newly created index
+   * @param index_oid index oid
+   * @param table_tuple_slot tuple slot
+   * @return Whether insertion was successful.
+   */
+  bool IndexCreateInsert(catalog::index_oid_t index_oid, storage::TupleSlot table_tuple_slot);
```

Comment:
See previous comment about this function's name. Also doesn't the `StorageInterface` already provide a way to insert into indexes?

Reply:
The one provided in the StorageInterface makes use of table_redo_->GetTupleSlot(), but the tuple slot comes from the translator in this case, so curr_index_->Insert is called directly